class.AttachmentPreviewPlugin.php

```diff
@@ -120,18 +120,20 @@ function bootstrap() {
 // Assuming that other plugins want to inject an element or two..
 // Provide a connection point to the attachments.wrapper
         Signal::connect(self::signal_id, function ($object, $data) {
-            $self::$instance->debug_log("Received connection from %s", get_class($object));
+            if (version_compare(PHP_VERSION, '5.4.0') >= 0)
+                $self::$instance->debug_log("Received connection from %s", get_class($object));
 // 
 // Assumes you want to edit the DOM with your structures, and that you've read the docs.
 // Just save them here until the page is done rendering, then we'll make all these changes at once:
-            self::$foreign_elements[get_class($object)] = $data;
+            AttachmentPreviewPlugin::$foreign_elements[get_class($object)] = $data;
```

Comment:
its selft of course, you can edit the pull due has the "edit by authors" flag enabled